### ISSUE TITLE
Cannot pan or scroll within element of webpage on some sites

### ISSUE THREAD

**Issue by freebrowser1** [fork-maintainers/iceraven-browser#795]
Some webpages have inner iframes but the user cannot scroll inside these frames.
Here an example: http://apolloarchive.com/apollo_gallery.html .
Open that page and click on any Apollo mission, e.g. Apollo 11.

Then the bottom half of the browser will show a clickable list of images or other media. But the user cannot scroll in this frame. Other (desktop) browsers can scroll in this frame.

Another example: https://wikisky.org/ (no frames).

This is a photograpic map of the sky which opens with an image of the Andromeda galaxy M31. Regardless of the object shown, panning is not possible. I tried with two fingers, holding a finger for a second and then swipe, but panning did not occur.

Other browsers allow panning the sky viewer.

**Comment by akliuxingyuan** [comment]
> Here an example: http://apolloarchive.com/apollo_gallery.html .

this one works for me

> Another example: [wikisky.org](https://wikisky.org/) (no frames).

I test Firefox for android and Chrome for android, both failed to panning, but I think the problem mostly is website itself, become other websites like map.google.com/what3workds.com work fine.

**Comment by freebrowser1** [comment]
> > Here an example: http://apolloarchive.com/apollo_gallery.html .
> 
> this one works for me
> 
> > Another example: [wikisky.org](https://wikisky.org/) (no frames).
> 
> I test Firefox for android and Chrome for android, both failed to panning, but I think the problem mostly is website itself, become other websites like map.google.com/what3workds.com work fine.
No, that is absolutely not the case: on a regular browser such as Chrome or Firefox on a computer, this site works normally. So it is a restriction in Android browsers.

**Comment by akliuxingyuan** [comment]
> No, that is absolutely not the case: on a regular browser such as Chrome or Firefox on a computer, this site works normally. So it is a restriction in Android browsers.

I already said other websites like map.google.com/what3workds.com work fine, wikisky itself doesn't have logic to handle event on android browser.

**Comment by freebrowser1** [comment]
> > No, that is absolutely not the case: on a regular browser such as Chrome or Firefox on a computer, this site works normally. So it is a restriction in Android browsers.
> 
> I already said other websites like map.google.com/what3workds.com work fine, wikisky itself doesn't have logic to handle event on **android browser**.
Exactly, that is the issue. An Android browser should not work differently on a particular web page than a Windows / Linux / macOS browser, and not at all when it is the same browser (Firefox).
